request/request.d.ts

```diff
@@ -13,57 +13,54 @@ declare module 'request' {
 	import http = require('http');
 	import FormData = require('form-data');
 	import url = require('url');
+	import fs = require('fs');
 
-	export = RequestAPI;
+	namespace request {
+		export interface RequestAPI<TRequest extends Request, TOptions extends OptionalOptions> {
```

Comment:
Would it be better to leave the RequestAPI untemplated and then override the functions definitions in RequestPromiseAPI?  I definitely agree with moving this to a interface though, the module with function exports isn't very flexible.